packages/framework/src/Model/Cart/Item/CartItem.php

```diff
@@ -86,7 +86,7 @@ public function __construct(
      */
     public function changeQuantity(int $newQuantity): void
     {
-        if (filter_var($newQuantity, FILTER_VALIDATE_INT) === false || $newQuantity <= 0) {
+        if (filter_var($newQuantity, FILTER_VALIDATE_INT) == false || $newQuantity <= 0) {
```

Comment:
Can you please explain this to me? I thought strict comparison is a better comparison :D
Also, if the point of this code is only to check if the value is int, I suggest to use `Webmozart\Assert\Assert::integer()`

Reply:
Strict comparison is better if you want compare types, but in this case you compare int and bool and in strict mode is this always false.

:+1: for `assert` 4608af90f54601f2c4ea0794c28838971dd21d8e